### PR TITLE
7368 accessibility widgets2of2 5 2 cherrypick

### PR DESCRIPTION
**Pull request by phudson-he**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change
Changes have been done to ensure [WCAG 2.1 requirements](https://www.w3.org/TR/WCAG21/#requirements-for-wcag-2-1) have been met for accessibility guidelines.

Details of the changes are as follows:

arches/app/templates/views/components/widgets

- radio-boolean.htm : made keyboard operable
- radio.htm : made keyboard operable
- resource-instance-select.htm : added aria-labels and made keyboard operable
- rich-text.htm : no change - cannot make accessible as a third party plugin is being used called ckeditor
- select.htm : added aria-labels
- switch.htm : made keyboard operable and made subtitle text color accessible

### Issues Solved

#

### Checklist

-   [x] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: Historic England
*   Found by: @phudson-he
*   Tested by: @phudson-he
*   Designed by: @phudson-he

### CODE REVIEW

**Comment by phudson-he** [comment]
@robgaston - This has now has dev/6.0.x merged in so includes accessibility mode/css. Tests and checks all passed.

**Comment by csmith-he** [comment]
@robgaston @aarongundel 

Hi both.  I've been asked to push forward outstanding Accessibility pull requests while @phudson-he is focusing on other Accessibility work.

I think for the older PRs I'll mostly be merging in dev/6.1.x changes (and resolving any conflicts) so they can be merged into dev/6.1.x as the base branch.  I know there are some queries with the newer PRs so I'll liaise with Paul to get those questions answered so we can resolve any outstanding issues.

Would it be possible for you to review this PR please?  Many thanks.

File: arches/install/arches-templates/project_name/templates/index.htm [hunk 1]
```diff
@@ -69,7 +69,6 @@
                             <h1>{% blocktrans %}Arches | {{version}}{% endblocktrans %}</h1>
                         </div>
                     </div>
-                </div>
```
Comment (aarongundel) [inline arches/install/arches-templates/project_name/templates/index.htm:72]: By my count, this div removal causes an imbalance.  It's worth noting that this div is also present in the main htm file.

Comment (aarongundel) [inline arches/install/arches-templates/project_name/templates/index.htm:72]: By my count, this div removal causes an imbalance.  It's worth noting that this div is also present in the main htm file.

**Review by aarongundel — CHANGES_REQUESTED** [review]
Thanks for this PR!  Looks pretty good, just a couple of minor points to discuss.

File: arches/app/media/css/index.css [hunk 2]
```diff
@@ -627,7 +643,7 @@ footer .footer-links {
     z-index: 100;
     color: #fff;
     background: #2a2a2a;
-    border: 1px solid #fff;
+    border: 3px solid #fff;
```
[the remark this answers is not in the document]
Comment (phudson-he) [reply arches/app/media/css/index.css]: I have changed this back to 1px as this is now in accessibility.css. The change is to make the border more visible for users with visual impairments, bearing in mind the colours of the background image it sits on top of.

File: arches/app/media/css/index.css [hunk 3]
```diff
@@ -63,6 +63,17 @@ div.collapse.navbar-collapse.navbar-ex1-collapse > div > ul {
     padding: 18px 20px;
 }
 
+.navbar-toggle {
```
[the remark this answers is not in the document]
Comment (phudson-he) [reply arches/app/media/css/index.css]: Now in accessibility.css

File: arches/app/media/css/index.css [hunk 4]
```diff
@@ -427,6 +439,10 @@ ul.cons-splash-list > li > a:focus, ul.cons-splash-list > li > a:hover {
     font-size: 20px;
 }
 
+.app-info-block-header {
```
[the remark this answers is not in the document]
Comment (phudson-he) [reply arches/app/media/css/index.css]: Removed duplicate app-info-block-header

[the remark this answers is not in the document]
Comment (phudson-he) [reply arches/app/media/css/index.css]: Removed duplicate app-info-block-header

File: arches/install/arches-templates/project_name/templates/index.htm (hunk 1 above)
Comment (phudson-he) [reply arches/install/arches-templates/project_name/templates/index.htm]: Added missing close div

File: arches/app/templates/views/components/widgets/radio-boolean.htm [hunk 5]
```diff
@@ -28,11 +28,11 @@
     {% trans "Default Value" %}
 </div>
 <div class="col-xs-12" style="display:flex; flex-direction: column; height: 50px">
-  <div style="pad-hor">
-      <label data-bind="css: { 'active': defaultValue() === true }, click: function(e){setDefaultValue(true)}" class="form-radio form-normal form-text">
+  <div class="pad-hor">
+      <label data-bind="css: { 'active': defaultValue() === true }, click: function(e){setDefaultValue(true)}" class="form-radio form-normal form-text" tabindex="0" onkeyup="if(event.which == 13 || event.keyCode == 13){ $(this).trigger('click'); }">
```
[the remark this answers is not in the document]
Comment (phudson-he) [reply arches/app/templates/views/components/widgets/radio-boolean.htm]: keyup event added to VM

[the remark this answers is not in the document]
Comment (phudson-he) [reply arches/app/templates/views/components/widgets/radio-boolean.htm]: keyup event added to VM

**Comment by phudson-he** [comment]
@aarongundel All requests resolved

**Pull request closed without merging**